ksql-common/src/main/java/io/confluent/ksql/util/timestamp/TimestampExtractionPolicyFactory.java

```diff
@@ -31,9 +33,18 @@ public static TimestampExtractionPolicy create(
       final LogicalSchema schema,
       final Optional<String> timestampColumnName,
       final Optional<String> timestampFormat
+  ) {
+    return create(schema, timestampColumnName, timestampFormat, new FailOnInvalidTimestamp());
+  }
+
+  public static TimestampExtractionPolicy create(
+      final LogicalSchema schema,
+      final Optional<String> timestampColumnName,
+      final Optional<String> timestampFormat,
+      final TimestampExtractor defaultTimestampExtractor
```

Comment:
```suggestion
      final TimestampExtractor timestampExtractor
```

nit: nothing default about it :D